docker/light-docker-compose.yml

```
@@ -3,14 +3,14 @@ version: '3'
 services:
   light0:
     container_name: light0
-    image: "ghcr.io/jbowen93/celestia-node:v0.1.2"
+    image: "ghcr.io/celestiaorg/celestia-node:v0.2.0"
```

this won't work due to a bug https://github.com/celestiaorg/celestia-node/issues/510

I propose we change the version for light as this :
```suggestion
    image: "ghcr.io/celestiaorg/celestia-node::sha-c5ee765@sha256:0c53fa3c8051a46a51d84a295edaa2621caa119c63feca48313595d6b9a765f0
```